tclint.toml

```diff
@@ -0,0 +1,11 @@
+exclude = [
+    'third_party',
+    'siliconcompiler/tools/yosys/syn_strategies.tcl'
```

Comment:
too messy to include?

Reply:
yeah, it's very closely adapted from another file and I don't think it's very well-tested, so didn't want to hack it up. I'll add a comment.